### DISCUSSION TITLE
Segmentation Fault with No errors in Shell or Logfile

### DISCUSSION THREAD

**Discussion by rskeys1** [srsran/srsRAN_Project#237] [General Help]
We have a new installation of srsRAN that appears to be communicating to our 5G core and our RU however after getting past the Fronthaul configuration (and fixing some PRACH / PUCCH frequency occasion collision settings in the config file), we're now getting a segmentation fault with no clue as to what happened since the logfile (path specified in config file) and shell show no information.

Is it possible to turn on any additional debugging to get some insight as to what is happening or is the only next step starting gdb and setting breakpoints in code?

Additionally, is there a complete TDD config file available as a reference? Another possibility for our segmentation fault may be that the gNB and frame configuration may not be fully specified in our config file but it would be good to start with a known good working reference TDD config file if one is available.

**Comment by ismagom** [comment]
The best starting point is to share the config file, log file and console output. We'll see what the next step it. Thanks
